### PR TITLE
chore: update README

### PR DESCRIPTION
**Pull request by chunshao90**
## Rationale
related https://github.com/apache/incubator-horaedb/issues/1342
Add quick start to `README` to help users experience horaedb better.

## Detailed Changes
Add quick start to `README`.

## Test Plan
No need.

### CODE REVIEW

**Comment by caicancai** [comment]
close: https://github.com/apache/incubator-horaedb/issues/1342

**Comment by caicancai** [comment]
Updating README was supposed to be my job in the beginning, but thanks for helping me with this part of the work, although it is a trivial small job, but I feel that it is not a good sign, hope that it will not happen in the future, it may affect the experience of other developers. Of course I didn't mean any offence, I just offered my opinion and suggestions, and I also hope that the community of horaedb will become better and better

File: README.md
Comment (caicancai) [inline README.md:26]: Create

**Comment by tanruixiang** [comment]
> Updating README was supposed to be my job in the beginning, but thanks for helping me with this part of the work, although it is a trivial small job, but I feel that it is not a good sign, hope that it will not happen in the future, it may affect the experience of other developers. Of course I didn't mean any offence, I just offered my opinion and suggestions, and I also hope that the community of horaedb will become better and better

@caicancai  I'm sorry that the core team members caused you a bad experience. Perhaps being new to the incubator caused some chaos. But I noticed that this pr still has imperfections, as you have suggested. Maybe you can use the suggestion mode to create the review comment(detail can follow https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request), after that I can merge your review as a commit, and you will be the co-author. Thank you for your attention in this issue, you deserve to be the co-author of this pr.

**Comment by chunshao90** [comment]
> Updating README was supposed to be my job in the beginning, but thanks for helping me with this part of the work, although it is a trivial small job, but I feel that it is not a good sign, hope that it will not happen in the future, it may affect the experience of other developers. Of course I didn't mean any offence, I just offered my opinion and suggestions, and I also hope that the community of horaedb will become better and better

Sorry for bringing you a bad experience. Since the repo was recently transferred to apache org, I found that the README was a little unfriendly to new users, so I simply added the quick start section. The current version is not perfect yet, you can continue to optimize it according to your ideas.

**Comment by chunshao90** [comment]
> close: #1342

I hadn't noticed this issue before. The changes I made in this PR are quite simple. You can continue to optimize it according to your ideas.

**Comment by caicancai** [comment]
> > Updating README was supposed to be my job in the beginning, but thanks for helping me with this part of the work, although it is a trivial small job, but I feel that it is not a good sign, hope that it will not happen in the future, it may affect the experience of other developers. Of course I didn't mean any offence, I just offered my opinion and suggestions, and I also hope that the community of horaedb will become better and better
> 
> I didn't notice the related issue. Sorry for bringing you a bad experience. Since the repo was recently transferred to apache org, I found that the README was a little unfriendly to new users, so I simply added the quick start section. The current version is not perfect yet, you can continue to optimize it according to your ideas.

Thank you for your patient reply. Due to my busy work recently, I did not follow up this issue in time, for which I am sorry. From this event, I can feel that incubator-horaedb is a good community and I am looking forward to making more contributions in the future

**Review by tanruixiang — APPROVED** [review]
LGTM

**Comment by tanruixiang** [comment]
Hi @caicancai, Already set you as co-author. docker image has been made, this pr is about to merge, if you have any modifications you are welcome to make them in the new pr.

**Comment by tanruixiang** [comment]
@ShiKaiWi Please help to review the docker images.

**Comment by ShiKaiWi** [comment]
> @ShiKaiWi Please help to review the docker images.

It works.

**Review by tanruixiang — APPROVED** [review]
LGTM

**Pull request merged**
